src/adapters/native.js

```diff
@@ -0,0 +1,95 @@
+var mkFhir = require('../fhir');
+
+// Fetch Helper JSON Parsing
+function parseJSON(response) {
+
+  return response.text()
```

Comment:
I have added a source comment, but it is because response.json() throws an error when the body is empty, such as the result from a DELETE operation